types/setting.go

```diff
@@ -423,6 +426,18 @@ var (
 			string(NodeDownPodDeletionPolicyDeleteBothStatefulsetAndDeploymentPod),
 		},
 	}
+
+	SettingDefinitionAllowDrainingNodeWithLastHealthyReplica = SettingDefinition{
+		DisplayName: "Allow Draining The node With The Last Healthy Replica",
```

Comment:
Weird CamelCase.

`Allow Node Drain with the Last Healthy Replica`.

Reply:
Thank you. Fixed